Echo server example using cne and libpnet.

manojgop opened the pull request:
- Example echo server implementation using cne and libpnet library.
- CNE receives and sends layer 2 packets using AF-XDP socket.
- libpnet receives and sends layer 2 packets using AF_PACKET.
- This application requires a load generator (s/w or h/w) to send packets
  to required n/w interface.

Signed-off-by: Manoj Gopalakrishnan <manoj.gopalakrishnan@intel.com>

KeithWiles reviewed (changes requested):
The code looks reasonable.

Maybe a bit more comments in the code could help with readability, please comment some before merging.

Maybe a comment at the top of the file explaining some of the overall code flow.

manojgop commented:
> The code looks reasonable.
> 
> Maybe a bit more comments in the code could help with readability, please comment some before merging.
> 
> Maybe a comment at the top of the file explaining some of the overall code flow.

I've added more comments and updated PR

KeithWiles reviewed (approved):
Thanks LGTM
